content/knowledge/flare_architecure.md

```diff
@@ -0,0 +1,152 @@
+# Flare Architecture
+
+This section contains information about some parts of the Flare Network.
+In addition, it also contains some information regarding the Avalanche platform, since Flare adopted some of its parts within its ecosystem.
+
+## Avalanche Platform
+
+Avalanche is an open-source platform for launching decentralized applications and enterprise blockchain deployments in one interoperable, highly scalable ecosystem.
+Avalanche is the first decentralized smart contracts platform built for the scale of global finance, with near-instant transaction finality.
+Ethereum's developers can quickly build on Avalanche as Solidity works out-of-the-box.
+
+A key difference between Avalanche and other decentralized networks is the consensus protocol.
+The Avalanche protocol employs a novel approach to a consensus to achieve its strong safety guarantees, quick finality, and high throughput without compromising decentralization.
+
+The purpose of the Avalanche platform:
+
+* creation of subnets and blockchains tied to the main Avalanche network for general and corporate purposes;
+* creation and hosting of decentralized applications;
+* creation of smart assets (for example, new financial derivatives and DeFi solutions).
+
+### Subnets
+
+Everything on Avalanche is a _subnet_, and every chain is part of a subnet.
+A Subnet, or Subnetwork, is a dynamic set of validators working together to achieve consensus on the state of a set of blockchains.
+Each blockchain is validated by exactly one Subnet.
+A Subnet can validate arbitrarily many blockchains.
+Before any of the Subnets can become a member of the network, it must stake some AVAX tokens.
+Clients interact with Avalanche through API calls to nodes (each chain has its [own API](https://docs.avax.network/build/avalanchego-apis/README)).
+
+There is a special subnet called the _Primary Network_, which validates Avalanche's built-in blockchains.
+A node may be a member of arbitrarily many Subnets and must be part of the primary network.
+The Primary Network contains three build-in blockchains: P-Chain, C-Chain, and X-Chain.
+
+In total, the Avalanche ecosystem (as of the end of December 2021) operates 13 subnets and 10 blockchains (three main plus 7 minor ones: Dtehereum, Btehereum, C and X-chain PoW, and others).
```

Comment:
Since the number of subnets is higher than number of blockchains, what do the extra subnets do exactly? 
Or in other words, what else do subnets do apart from validating blockchains?

Reply:
Interestingly they just exist there and I guess do nothing https://avascan.info/blockchains?subnet=2TSYtJmLdmBJCTcEP6ZeQ3d7urg1GbrKtVMMdd7nmz46z22PSC